src/style.css

```diff
@@ -0,0 +1,55 @@
+body {
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.logo__link {
+  margin: 8px 0 8px 50px;
+
+}
+
+.nav {
+  margin-right: 50px;

```

Comment:
Instead of margins for logo and nav, you could use padding: 0 50px for your header